### DISCUSSION TITLE
Check if user is authenticated without making any HTTP requests

### DISCUSSION THREAD

**Discussion by Pacheco95** [adonisjs/core#1657] [Help]
I'm using [Web Guard](https://preview.adonisjs.com/guides/auth/web-guard) as authentication method and login/logout is working well but I need to make private routes in my React app and I don't know how to do this using the cookie based authentication without making HTTP request every time I reload a private route. Is there a way to make it simpler?

**Comment by thetutlage** [comment]
Theoretically speaking. How is that possible? Even if you use token based authentication, you still have to check with the backend to ensure that token is still valid and not been revoked/expired

**Reply by Pacheco95** [reply]
I really have no idea. I think that may have a way to use the same backend validation logic in frontend. But I undestand the need for a backend request even with an http request for each private route access
